Модуль 11

Stas2242 opened the pull request:
- Добавлен localStorage для даты и заголовка
- добавлена логика для кнопки сброса

Stas2242 committed:
Модуль 11

- Добавлен localStorage для даты и заголовка
- добавлена логика для кнопки сброса

KazaninaYana commented on index.js:
```diff
@@ -15,26 +15,28 @@ let timerId = null
 
 function timerStart () {
-    timerDeadline = inputDate.value
+    const localdate = localStorage.setItem('date', inputDate.value);
```
а нужна ли нам эта переменная здесь? Она что-то содержит? Где-то используется?

Stas2242 replied:
В timeStart убрал переменную timerDeadline. Чтобы работал второй if заменил timerDeadline на inputDate.value

KazaninaYana commented on index.js:
```diff
@@ -67,11 +70,31 @@ function timerReset() {
+
+    if (!localdate && !localTitle) {
+        return;
+    } else {
```
так как в прошлом if стоит return, то  else мы можем уже и не писать, код и так будет выполняться только если не попадем в if) Этот прием еще ранний выход назвают

Stas2242 replied:
Убрал Else

KazaninaYana commented on index.js:
```diff
@@ -67,11 +70,31 @@ function timerReset() {
+    if (!localdate && !localTitle) {
+        return;
+    } else {
+        timerDeadline = localdate
```
точка с запятой

KazaninaYana replied:
в строке 90 тоже

Stas2242 replied:
Проставил ";"

KazaninaYana commented on index.js:
```diff
@@ -67,11 +70,31 @@ function timerReset() {
 }
 
+const localStorageCheck = () => {
+    const localdate = localStorage.getItem('date')
```
localdate это по сути два слова, значит второе должна начинаться с большой буквы (как localTitle)

Stas2242 replied:
Исправлено

Stas2242 committed:
Update index.js

В timeStart убрал переменную timerDeadline. Чтобы работал второй if заменил timerDeadline на inputDate.value

Stas2242 committed:
Update index.js

- Удалена лишняя переменная в стр 18
